libs/designer/src/lib/ui/mcp/panel/connector/tabs/connectionsTab.tsx

```diff
@@ -14,8 +14,8 @@ export const connectionsTab = (
 ): McpPanelTabProps => ({
   id: constants.MCP_PANEL_TAB_NAMES.CONNECTIONS,
   title: intl.formatMessage({
-    defaultMessage: 'Create connection',
-    id: 'Cosbik',
+    defaultMessage: 'Create connections',
+    id: '3n9+Ka',
```

Comment:
But we are only creating connection for single connector no? then why connections?